pkg/crc/oc/oc.go

```diff
@@ -0,0 +1,30 @@
+package oc
+
+import (
+	"path/filepath"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	crcos "github.com/code-ready/crc/pkg/os"
+)
+
+type OC struct {
+	OcBinaryPath   string
+	KubeconfigPath string
+}
+
+func GetOC(machineName string) OC {
+	oc := OC{
+		OcBinaryPath:   filepath.Join(constants.OcCacheDir, constants.OcBinaryName),
+		KubeconfigPath: filepath.Join(constants.MachineInstanceDir, machineName, "kubeconfig"),
```

Comment:
Fwiw, we should have some helpers to find where the kubeconfig file and the oc file were stored by crc, I don't think it's good to duplicate their exact location each time we need it. However, the problem is preexisting with eg the `machines` dir, so this can be solved at a later time.

Reply:
'were' stored? I do not think we need to figure this out, but have consistency about what we will do.

It would be best to ALWAYS use the version from the running instance, which therefore means we need to copy the kubeconfig